cfme/tests/containers/test_container_provider_crud.py

```diff
@@ -0,0 +1,67 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.common.provider_views import ContainerProvidersView
+from cfme.containers.provider import ContainersProvider
+from cfme.fixtures.provider import setup_or_skip
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+from cfme.utils.blockers import BZ
+from cfme.utils.update import update
+
+
+pytestmark = [
+    pytest.mark.tier(1),
+    test_requirements.containers,
+    pytest.mark.provider(
+        [ContainersProvider],
+        scope='function',
+        selector=ONE_PER_VERSION),
+    pytest.mark.meta(
+        blockers=[BZ(1713504, forced_streams=['5.11'])]
+    ),
+]
+
+
+def test_container_provider_crud(request, appliance, has_no_providers, provider):
+
+    """
+    Polarion:
+        assignee: juwatts
+        caseimportance: critical
+        casecomponent: Containers
+        initialEstimate: 1/6h
+    """
+
+    setup_or_skip(request, provider)
```

Comment:
I don't think you want setup_or_skip, if it fails to add the provider, that's a test failure, not skip.

I would call provider.create() directly so that it gets added through the UI.